project/build.properties

```
@@ -1 +1 @@
-sbt.version=1.10.3
+sbt.version=1.10.5
```

```suggestion
sbt.version=1.10.6
```

We should probably use the newest sbt version whenever possible for dogfooding purpose?